### DISCUSSION TITLE
现在微信好像走x64版本了, 怎么找特征码啊

### DISCUSSION THREAD

**Discussion by xx10n31y** [huiyadanli/RevokeMsgPatcher#618] [General]
用x32dbg找不到wechat.exe了, 用x64dbg可以找到, 但好像按照Wiki的方法不太行得通了, 现在应该怎么用x96dbg做补丁

**Comment by huiyadanli** [comment]
用x64dbg，一样的步骤，不过老的特征失效了

**Reply by xx10n31y** [reply]
还是从wechatwin.dll里面搜索字符串吗, 还是revokemsg字符串吗

**Reply by huiyadanli** [reply]
是的

**Reply by xx10n31y** [reply]
还是第一个 "revokemsg" 字符串然后上一行je => jmp 吗. 我试了一下,好像不是了
